R/data-converters.R

```diff
@@ -348,13 +348,13 @@ enw_incidence_to_cumulative <- function(obs, by = NULL) {
 #' @export
 #' @family dataconverters
 #' @examples
-#' nat_hosp <- germany_covid19_hosp[location == "DE"][age_group %in% "00+"]
+#' nat_hosp <- germany_covid19_hosp[location == "DE"][age_group == "00+"]
 #' enw_aggregate_cumulative(nat_hosp, timestep = "week")
 enw_aggregate_cumulative <- function(
   obs, timestep = "day", by = NULL,
   min_reference_date = min(obs$reference_date, na.rm = TRUE), copy = TRUE
 ) {
-  stopifnot("The data already has a timestep of a day" = !timestep %in% "day")
+  stopifnot("The data already has a timestep of a day" = !timestep == "day")
```

Comment:
```suggestion
  stopifnot("The data already has a timestep of a day" = timestep != "day")
```

There is a linter coming for this some it might be worth getting ahead of it here?